Go/Go.sublime-settings

```diff
@@ -0,0 +1,11 @@
+{
+  // Prefer tabs because gofmt enforces tabs.
+  "translate_tabs_to_spaces": false,
```

Comment:
This is certainly debatable, but I don't have insight on whether people write Go without gofmt.